evap/staff/forms.py

```diff
@@ -31,7 +31,24 @@ class ImportForm(forms.Form):
 
 
 class UserImportForm(forms.Form):
-    excel_file = forms.FileField(label=_("Excel file"))
+    excel_file = forms.FileField(label=_("Import from Excel file"), required=False)
+    course = forms.ModelChoiceField(Course.objects.all(), required=False, label=_("Copy from Course"))
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        choices = [('', '---------')]
```

Comment:
is this still necessary?

also, please add some comment explaining the grouping thing